### PR TITLE
Changes used at the Aalborg workshop

### PR DESCRIPTION
**Pull request by bast**
This PR depends on approval of https://github.com/coderefinery/reproducible-research/pull/66. Marked as draft until the other PR is approved.

### CODE REVIEW

**Comment by bast** [comment]
Ready for review, but depends on https://github.com/coderefinery/reproducible-research/pull/66.

**Comment by bast** [comment]
Once merged, please remove https://github.com/coderefinery/word-count-aalborg.

File: Makefile [hunk 1]
```diff
@@ -1,15 +1,26 @@
-# Build both steps required for executing the character count example
-all: processed_data/abyss.dat results/abyss.png results/results.txt
+SRCDIR := data
```
Comment (rkdarst) [inline Makefile:2]: I suggest a glossary of the used Make variables at the top of the file.  That would go a long way towards readability.

**Review by rkdarst — APPROVED** [review]
I haven't looked in great detail, but looks good to me.

File: Makefile (hunk 1 above)
Comment (bast) [reply Makefile:2]: Good idea - this is now done.

Comment (rkdarst) [reply Makefile:2]: What I meant was these variables:

- $%
- $@
- $<
- $(RM)

Maybe I'll make a edit with these

**Pull request merged**